tools/scripts/misc/bm_concat_logs.py

```diff
@@ -0,0 +1,25 @@
+import sys
+import datetime
```

Comment:
These next few files are random scripts I created while testing rtc-0, rbr resets, soft resets, capturing usb output. They are not polished and I can delete them from this PR. But I thought others/myself might be able to quickly modify, and use these again in the future and having them committed prevented "dirty" builds 😅.